package.json

```diff
@@ -41,7 +41,7 @@
     "codeclimate-test-reporter": "^0.4.0",
     "colors": "^1.1.2",
     "css-loader": "^0.26.1",
-    "d2": "^26.1.7",
+    "d2": "^26.1.12",
```

Comment:
Point to d2 26.1.7? Probably latest d2 version contains some bugfixes may be interested...